gerousiabot/discord_bot.py

```diff
@@ -0,0 +1,92 @@
+from typing import List
+
+from discord import Intents, Member, VoiceChannel, Client
+
+from gerousiabot import utils
+
+logger = utils.setup_logger()
+
+
+class GerousiaBot(Client):
+
+    def __init__(self, bot_token: str, **options):
+        intents = get_intents_needed_to_check_online_members()
+        logger.debug('Creating bot with with intents {}'.format(intents))
+
+        super().__init__(intents=intents, **options)
+        logger.debug('Creating bot with with valid token {}'.format(bot_token is not None))
+        self.bot_token = bot_token
+
+    async def on_ready(self):
+        """
+        Asynchronous event that fires once the bot has logged in and ready to work.
+        """
+        logger.info('Bot has logged in as {0.user}'.format(self))
+
+        server_id = int(utils.get_env_variable('GOOD_LOOKING_GAMERS_SERVER_ID'))
+        guild_voice_channels = await self.get_server_voice_channels(server_id)
+        members_in_voice_channels = await self.get_members_who_are_in_voice_channels(guild_voice_channels)
+
+        for member in members_in_voice_channels:
```

Comment:
Technically we don't need to print the members out. I added it in here for the proof on concept

Reply:
i'm aware but since we haven't decided where we going after i left it